css/amp.css

```diff
@@ -146,7 +146,12 @@ i-amp-sizer {
   /* Just state. */
 }
 
+
+/** Only show the last placeholder element */
 .-amp-element > [placeholder] {
+  display: none;
+}
+.-amp-element > [placeholder]:last-of-type {
```

Comment:
Unfortunately this doesn't really mean what I thought it meant. This only checks the tag so this wouldn't work as expected unless the placeholders are of the same tag type.

This breaks things, multiple placeholders will still show if they're of different types because the selector will match them all. I am trying to think of a way around this but so far coming empty. Let me know if you have ideas on how to deal with this.

cc/ @cramforce @erwinmombay @dvoytenko - see for more details #2830